### ISSUE TITLE
khorosjx.users.get_recent_logins() not making verify_core_connection() 

### ISSUE THREAD

**Issue by jeffshurtliff** [jeffshurtliff/khorosjx#1]
The `khorosjx.users.get_recent_logins()` function is missing the function call for `verify_core_connection()` which can result in the following `NameError` exception occurring:

``` python
Traceback (most recent call last):
  File "recurring/analyze_logins.py", line 50, in get_logins
    login_data = khorosjx.users.get_recent_logins(count, start_index)
  File "/home/shurtj/.local/lib/python3.6/site-packages/khorosjx/users.py", line 338, in get_recent_logins
    query_url = f"{base_url}/people?sort=lastLoggedIn&" + \
NameError: name 'base_url' is not defined
```

**Comment by jeffshurtliff** [comment]
A fix for the issue has been deployed in the pre-release version [khorosjx-1.6.0.dev0](https://pypi.org/project/khorosjx/1.6.0.dev0/).

**Comment by jeffshurtliff** [comment]
Confirmed that the issue is resolved in the stable version [khorosjx-1.6.0](https://pypi.org/project/khorosjx/1.6.0/), as demonstrated below.
``` python
>>> login_data = khorosjx.users.get_recent_logins()
>>> login_data[0]['id']
'14845'
>>>
```
